index.html

```diff
@@ -1,29 +1,183 @@
-        <li><form action="">
-            <input type="search" name="" id="" placeholder="Search">
-            <button><i class="fa fa-search"></i></button>
-        </form></li>
-        <li><a href=""><i class="fa fa-upload" ></i></a></li>
-        <li><button>Sign in</button></li>
-    </ul>
-    </nav>
+  <nav class="navbar">
+    <div class="navbar-items">
+      <i id="menu-icon" class="material-icons">menu</i>
+      <img class="logo" src="images/youtube.png">
+    </div>
+    <div class="navbar-items">
+      <div class="wrap">
+        <div class="search">
+          <input type="text" class="searchTerm" placeholder="Search">
+          <button type="submit" class="searchButton">
+            <i class="fa fa-search"></i>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="navbar-items">
+      <div class="notification">
+        <i id="upload-icon" class="fas fa-upload"></i>
+      </div>
+      <div class="notification">
+        <i id="apps-icon" class="material-icons">apps</i>
+      </div>
+      <div class="notification">
+        <i id="notification-icon" class="fas fa-bell"></i>
+      </div>
+      <div class="notification">
+        <i id="user-icon" class="far fa-user-circle"></i>
+      </div>
+    </div>
+  </nav>
+  <section class="container">
+    <section class="video">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/V74l_zS1x8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>            
+        <section class="description">
+          <section class="video-details">
+            <h1>Embedded Video Player</h1>
+            <section class="video-socials">
+              <section class="views">
+                <span class="count">
+                  1.1M views •
+                </span>
+                <span class="date">
+                  Aug 11, 2020
+                </span>
+              </section>
+              <div class="flex"></div>
+
+              <section class="socials-btn">
+                <i class="material-icons">thumb_up</i>
+                <span class="btn-value">5.6K</span>
+                <i class="material-icons">thumb_down</i>
+                <span class="btn-value">333</span>
+                <i class="material-icons">playlist_add</i>
+                <span class="btn-value">SAVE</span>
+                <i class="fa fa-share"></i>
+                <span class="btn-value">SHARE</span>
+              </section>
+              <i class="fa fa-ellipsis-h"></i>
+            </section>
+          </section>
+          <section class="channel-details">
+            <div class="channel">
+              <img class="channel-icon"src="images/channel.JPG" alt="Avatar">
+              <div class="channel-name">
+                <span style="font-size: 1;"><b>Microverse Projects</b></span>
```

Comment:
- [ ] For good coding practices, please do not use inline styling, as mentioned in the [article pointed by Pathwright](https://speckyboy.com/good-bad-css-practices/)